src/app/public/modules/lookup/lookup.component.ts

```diff
@@ -158,7 +190,8 @@ export class SkyLookupComponent
     private windowRef: SkyAppWindowRef,
     @Self() @Optional() ngControl: NgControl,
     private adapter: SkyLookupAdapterService,
-    @Optional() public inputBoxHostSvc?: SkyInputBoxHostService
+    @Optional() public inputBoxHostSvc?: SkyInputBoxHostService,
+    @Optional() public themeSvc?: SkyThemeService
```

Comment:
Just noticed we're using `?` here for theme service and not in other components. Should we make it consistent?

Reply:
I fixed this in the autocomplete component. The others are in the visual tests though and I verified that our precedence for the visual tests is to not list it as optional.